src/weekly/query.js

```diff
@@ -58,13 +58,15 @@ async function queryZuora (deliveryDate, config: Config) {
            ) OR
           (
             RatePlan.AmendmentType = 'RemoveProduct' AND 
-            Amendment.EffectiveDate > '${formattedDeliveryDate}' 
+            Amendment.EffectiveDate > '${formattedDeliveryDate}' AND
+            EffectiveStartDate <= '${formattedDeliveryDate}' 
```

Comment:
Just for future readers, this section is selecting **in**, still-active rate plan charges which have a future-dated (in respect to the fulfilment date) removal scheduled.